src/yz_entropy_mgr.erl

```diff
@@ -502,6 +503,7 @@ tick(S) ->
     S3 = lists:foldl(fun(_,SAcc) ->
                              maybe_poke_tree(SAcc)
                      end, S2, lists:seq(1,10)),
+    update_throttle(S3),
```

Comment:
I didnt even see it wasn't in the init. That makes sense, thought you were updating it immediately after the init_throttle. Makes sense.